packages/odyssey-react/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "@okta/odyssey-react",
+    "version": "0.7.0",
+    "description": "React components for Odyssey, Okta's design system",
+    "main": "src/index.js",
+    "repository": "https://github.com/okta/odyssey",
+    "author": "Okta, Inc.",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "classnames": "^2.3.1",
+        "react": "^17.0.1",
+        "react-dom": "^17.0.1"
```

Comment:
We should consider having these be peerDependencies so users can bring their own react version. https://stackoverflow.com/questions/26737819/why-use-peer-dependencies-in-npm-for-plugins